### PR TITLE
report create and download csv

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: reports/stores_report.csv
Comment (Taraslysak) [inline reports/stores_report.csv:1]: I think this file should be in gitignore

Comment (Taraslysak) [reply reports/stores_report.csv:1]: If we need it at all

**Review by Taraslysak — COMMENTED** [review]
Please remove redundant file

File: reports/stores_report.csv
Comment (varvarashcherbyna7) [reply reports/stores_report.csv:1]: ok

**Pull request merged**